plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotation.kt

```diff
@@ -84,7 +84,7 @@ class PointAnnotation(
     get() {
       val value = jsonObject.get(PointAnnotationOptions.PROPERTY_ICON_ANCHOR)
       value?.let {
-        return IconAnchor.valueOf(it.asString.uppercase())
+        return IconAnchor.valueOf(it.asString.toUpperCase(Locale.US))
```

Comment:
Why `US`? 😄 
Guess `Locale.getDefault()` should fit better

Reply:
actually specific US locale might be better. Have seen issues in the past with not using a specific locale. 
Will try finding that issue in old the repo.

Reply:
tbh in this current scenario almost sure it does not matter, but if default locale may cause issues, I'm totally fine with `US`.

Reply:
https://developer.android.com/reference/java/util/Locale.html 
> Note that there are many convenience methods that automatically use the default locale, but using them may lead to subtle bugs.

Reply:
I saw we have used `Locale.US` in other generated files as well, so used the same for now. but yeah we can change it with default locale everywhere in one go.

Reply:
No need, let's stick to `US` as @tobrun mentioned above that using default may lead to bugs.